app/templates/views/signin.html

```diff
@@ -13,8 +13,17 @@
 {% block maincolumn_content %}
 
 <div class="grid-row">
-  <div class="tablet:grid-col-12">
-
+  <div id="countdown-container" class="usa-alert usa-alert--warning width-full margin-bottom-4">
+    <div class="usa-alert__body">
+      <h4 class="usa-alert__heading">Login.gov is required by April 9, 2024</h4>
+      <p class="usa-alert__text">
+        You have <span id="countdown"></span> left to use Login.gov to sign in
+      </p>
+    </div>
+  </div>
+</div>
```

Comment:
Let's wrap this in the conditional check like the link below, so that we can hide this until we're ready to turn it on fully.  I think I have my nesting correct here with the `div`s, but please double check!

You'd just have to wrap this in the same `if` statement as below:

```
{% if login_gov_enabled %}
...
{% endif %}
```